services/notification/types/commitstatusdata.go

```diff
@@ -0,0 +1,33 @@
+package types
+
+import (
+	gitsource "agola.io/agola/internal/gitsources"
+	"agola.io/agola/internal/sqlg"
+	"agola.io/agola/internal/sqlg/sql"
+)
+
+type CommitStatusData struct {
+	sqlg.ObjectMeta
+
+	ProjectID    string       `json:"project_id"`
+	CommitStatus CommitStatus `json:"commit_status"`
+	CommitSHA    string       `json:"commit_sha"`
+	RunCounter   uint64       `json:"run_counter"`
+	Description  string       `json:"description"`
+	Context      string       `json:"context"`
+}
+
+type CommitStatus gitsource.CommitStatus
```

Comment:
Don't import gitsource here. This MUST be defined as a string since you're redefining the enum below.